style2.css

```diff
@@ -400,4 +452,365 @@
+  top: 3920px;
+  left: 135px;
+}
+
+.css-language {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 17px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 30px;
+  position: absolute;
+  top: 3920px;
+  left: 215px;
+}
+
+.ficon {
+  position: absolute;
+  left: 112px;
+  top: 4115px;
+}
+
+.boot {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+}
+
+.ruby-rails {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+}
+
+.r-spec {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 5px;
+  margin-right: 8px;
+}
+
+.Capybara {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 50px;
+  margin-right: 5px;
+}
+
+.Sele {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 15px;
+  padding-bottom: 15px;
+  padding-right: 10px;
+  padding-left: 10px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 5px;
+  margin-right: 8px;
+}
+
+.codekick {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-left: 8px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.github {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.codepen {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-right: 8px;
+}
+
+.gitlab {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+}
+
+.term {
+  background-color: white;
+  color: #36b37e;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  padding-right: 5px;
+  padding-left: 5px;
+  font-size: 18px;
+  line-height: 16px;
+  border-radius: 8px;
+  margin-left: 120px;
+}
+
+.projects {
+  width: 327px;
+  height: 363px;
+  color: #172b4d;
+  font-weight: bold;
+  font-size: 40px;
+  white-space: initial;
+  margin-left: 40px;
+  margin-top: 20px;
+}
```

Comment:
- [ ] Kindly make sure to stick to the provided Figma design. Please use the correct fonts and I think it would be better if you wrap this paragraph into a `footer` container and style it exactly as how it looks in Figma design. :wink:

   - ### Your design
    ![image](https://user-images.githubusercontent.com/87197412/186274743-18cb2f92-4560-44f8-acfb-d76b4c865da9.png)

   - ### Figma design
   

   ![image](https://user-images.githubusercontent.com/87197412/186274683-62bf3a29-3189-414e-b2ed-81fe47b3ced4.png)